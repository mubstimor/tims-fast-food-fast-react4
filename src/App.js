import React, { Component } from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';

//  Load components
import Navbar from './components/layout/Navbar';
import Login from './components/auth/Login';
import Register from './components/auth/Register';

class App extends Component {
  render() {
    return (
      <Router>
        <div className="App">
          <Navbar />
          <div className="container">
            <Route exact path="/" component={Login} />
            <Route path="/users/login" component={Login} />
            <Route path="/users/register" component={Register} />
          </div>
        </div>
      </Router>
    );
  }
}

export default App;
