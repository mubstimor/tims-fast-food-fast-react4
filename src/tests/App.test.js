import React from 'react';
import { shallow } from 'enzyme';

// Components
import App from '../App';

function setup() {
  const wrapper = shallow(<App />);
  return { wrapper };
}

describe('Welcome to APP', () => {
  it('Should load app', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
