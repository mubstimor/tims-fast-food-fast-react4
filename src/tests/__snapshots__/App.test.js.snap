// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome to APP Should load app 1`] = `ShallowWrapper {}`;
